lib/darjeelink/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Darjeelink
-  VERSION = '0.14.2'
+  VERSION = '0.14.3'
```

Comment:
I assume this is what we do with project. Could not find docs, so guessing this might be a Ruby 'convention'.

Reply:
This is related to the gem release and is detailed in (https://github.com/38degrees/darjeelink/blob/main/README.md)

Reply:
This also has  a different deploy to most of our apps as most of the code is in the gem.  Its therefore important to read the readme and follow the steps in it for deployment otherwise portkey won't get the updated gem

Reply:
Yup. Was going through the steps. 

Unfortunatly, you need to check everything in before your do them - `Once all necessary changes have made it in to master and you are ready to do a release you need to do these steps.`.